rule/add-constant.go

```diff
@@ -63,7 +63,7 @@ func (r *AddConstantRule) Apply(file *lint.File, arguments lint.Arguments) []lin
 }
 
 // Name returns the rule name.
-func (*AddConstantRule) Name() string {
+func (r *AddConstantRule) Name() string {
```

Comment:
What is the reason of adding it? It's unused. Here and in most of the other places you did that.

Reply:
ok maybe for Name() was not a good idea, i reverted changes